build/test.mk

```diff
@@ -74,8 +74,14 @@ test-functional-kubernetes: test-functional-kubernetes-noncloud ## Runs all Kube
 test-functional-kubernetes-noncloud: ## Runs Kubernetes functional tests that do not require cloud resources
 	CGO_ENABLED=1 $(GOTEST_TOOL) ./test/functional-portable/kubernetes/noncloud/... -timeout ${TEST_TIMEOUT} -v -parallel 5 $(GOTEST_OPTS)
 
-test-functional-shared: ## Runs shared functional tests
-	CGO_ENABLED=1 $(GOTEST_TOOL) ./test/functional/shared/... -timeout ${TEST_TIMEOUT} -v -parallel 10 $(GOTEST_OPTS)
+test-functional-shared: test-functional-corerp-noncloud test-functional-corerp-cloud ## Runs all Core RP functional tests (both cloud and non-cloud)
+	@echo "$(ARROW) Running corerp functional tests"
```

Comment:
Is this supposed to be blank? If this is here for historical purposes, when can it be removed?

Reply:
You mean it should be test-functional-corerp instead of test-functional-shared? This PR will still run against the old workflow in the main  branch which uses "shared" instead of "corerp". I will change the workflow in a separate PR once the "corerp" directory structure is checked in